contents/blog/gdpr-google-analytics-privacy-shield.md

```diff
@@ -0,0 +1,82 @@
+
+### May 24, 2016: The GDPR is approved by EU
+The GDPR, short for General Data Protection Regulation, becomes EU law. It's the most comprehensive data protection legislation in the world. Among its many directives, it stipulates that the personal data of any EU citizen cannot be exported to another nation that lacks equivalent data protection laws or contractual protections.   
+
+### July 12, 2016: The EU-US Privacy Shield is created
+In response to Schrems I, the EU and US negotiate the EU-US Privacy Shield, a replacement for the previous 'Safe Harbor' rules. It's designed to allow US companies to receive data from the EU while adhering to EU privacy laws.
+
+### January 25, 2017: President Donald Trump signs the "Enhanced Public Safety" executive order
+The new executive order demands that government agencies ensure their privacy policies "exclude persons who are not United States citizens or lawful permanent residents" from the protections of the Privacy Act. President Joe Biden would later repeal the executive order in January, 2021.
+
+### May 25, 2018: EU member states ratify the GDPR
+Just over two years after it was first approved by the EU, all member states have passed the GDPR in their national parliaments and the GDPR is now in force. National data protection authorities become responsible for enforcing the law in their countries.
+
+### July 16, 2020: European court rules the EU-US Privacy Shield invalid
+Over two years after a new complaint filed by Max Schrems in Ireland, the Court of Justice of the European Union (CJEU) rules the Privacy Shield [doesn't provide adequate protection](https://gdprhub.eu/index.php?title=CJEU_-_C-311/18_-_Schrems_II) for EU citizens. Existing Standard Contractual Clauses (SCC) allowing data transfer to the US are deemed inadequate.
+
+### August 17, 2020: Max Schrems files complaint with Austrian DPA
+Schrems alleges an Austrian website and Google violated the GDPR when a data subject visited a website on health topics. The complaint details how personal data, including user identifiers, IP address and browser details, were exported from Austria to the US where it could be subject to local law enforcement requests for access.
+
+### July 16, 2021: Amazon fined €746 million for GDPR breach
+Amazon [is fined](https://www.wired.co.uk/article/amazon-gdpr-fine) €746 million for failing to follow rules regarding its use of targeted advertising and proper consent. It's the largest GDPR fine to date.
+
+### January 13, 2022: Austrian authority rules Google Analytics violates GDPR
+The Austrian data authority partially upholds the complaint, ruling that the Austrian website did violate the GDPR. Google, as a data processor, isn't deemed liable.
+
+## What happens next?
```

Comment:
This would be a great place to introduce a link and screenshot to isgoogleanalyticsillegal.com potentially?